### PR TITLE
feat(sks): add default dns domain and record

### PR DESCRIPTION
**Pull request by hbollon**
I added a default dns domain ressource and a wildcard dns record one to avoid cert-manager issues on blue/green deployment.
@raphink

### CODE REVIEW

File: modules/sks/exoscale/main.tf [hunk 1]
```diff
@@ -121,6 +121,18 @@ resource "exoscale_security_group_rule" "all" {
...
+resource "exoscale_domain" "this" {
+  name = var.base_domain
+}
+
+resource "exoscale_domain_record" "wildcard" {
+  domain      = data.exoscale_domain.this.id
+  name        = format("*.apps.%s", var.cluster_name)
+  record_type = "CNAME"
+  ttl         = 300
+  content     = data.exoscale_domain_record.this.hostname
```
Comment (Vampouille) [inline modules/sks/exoscale/main.tf:133]: Where this datasource is defined ?

**Comment by raphink** [comment]
I think this will fail if no `base_domain` provided. There should be a condition.

**Review by Vampouille — DISMISSED** [review]

**Comment by hbollon** [comment]
I replaced `var.base_domain` by `local.base_domain` so it's should be good now @raphink

**Comment by Vampouille** [comment]
> I think this will fail if no `base_domain` provided. There should be a condition.

@raphink I think we can use the local `base_domain` instead of the variable ?

**Review by Vampouille — DISMISSED** [review]

**Review by yann-soubeyrand — APPROVED** [review]

**Pull request merged**